.travis.yml

```diff
@@ -1,16 +1,27 @@
 language: go
 
+# cache:
```

Comment:
Can we have TODO comment here. Commented lines which doesn't have any purpose doesn't look good at least to me.